packages/api/src/transaction/transaction.ts

```diff
@@ -35,3 +37,27 @@ export const getTransactionById = (
     authorization: true,
   })
 }
+
+export const batchSend = async (
+  client: Client,
+  transactions: Array<SendArgs>,
+) => {
+  try {
+    const body = transactions.map(({ tokenId, ...transaction }) => ({
```

Comment:
not part of the scope of this PR, but we're introducing an unnecessary loop here to solve for a naming convention in this repository that is unconventional in terms of standards across other repositories.

Generally, we have used those naming conventions for different identifiers:
- tokenID | a tokens uuid
- tokenId | a tokens serial ID (postgres)

This hasn't been a problem for this repository, but now we are modifying the behavior of our code to account for a naming convention. Something to think about